File: app/src/main/java/com/alex/testeandroid/presentation/MainActivity.java
package com.alex.testeandroid.presentation;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.alex.testeandroid.R;

/**
 * Created by Alex on 27/08/18.
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);
    }
}

File: SantanderTeste/app/src/main/java/br/com/rafael/santanderteste/presentation/MainActivity.kt
package br.com.rafael.santanderteste.presentation

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import br.com.rafael.santanderteste.R
import br.com.rafael.santanderteste.domain.CellCatalog

class MainActivity : AppCompatActivity(), FormContract.View {

        override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

            var presenter = FormPresenter()
            presenter.setView(this)
            presenter.loadCellsList()
    }

    override fun showCellsList(cellCatalog: CellCatalog) {

    }

    override fun showLoadingCells() {

    }

    override fun showErrorToLoadingCells() {

    }
}
